src/script/cpu-map.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+# !
+# ! Usage: ./cpu-map.sh -p <process_PIDs comma separated> -n <process_Name> -g (optional) <by-group>
+# !
+# ! Set the CPU affinity for a list of running processes
+# ! Ex.: ./cpu-map.sh -n scylla
+# !      ./cpu-map.sh -n scylla -g
+# !      ./cpu-map.sh -p "1362, 1049"
+# !      ./cpu-map.sh -p 32 -g
```

Comment:
This looks like it's lifted from seastar.git scripts/seastar-cpu-map.sh.  That's fine, but let's update the examples to suggest how to use it specifically with crimson-osd or ceph-osd?